src/main/java/com/mineinabyss/mobzy/ecs/goals/mobzy/hostile/ThrowItemsGoal.kt

```diff
@@ -40,7 +42,9 @@ class ThrowItemsBehavior(
         minChaseRad,
         minThrowRad,
         yOffset,
-        cooldown
+        projectileSpeed,
+        projectileRandomAngle,
```

Comment:
If you'd like to use another word, perhaps angular diameter works. It's used to describe the apparent size of objects in the sky.